### PR TITLE
Minor HTML updates to index.html.erb

### PR DESCRIPTION
**Pull request by maxkaran**
## Proposed Change
Changed some wording on the index.html.erb page

## Pull Request Checklist:

  - [ ] My submission passes all tests.
  - [ ] I have lint my code locally prior to submission.
  - [ ] I have written new tests for my core changes (where applicable).
  - [ ] I have referenced all useful information (issues, tasks, etc) in the pull request.

### CODE REVIEW

File: app/views/job_posting_questions/index.html.erb [hunk 1]
```diff
@@ -43,8 +43,8 @@
           Recommmended Questions:
         </p>
         <ul>
-          <li>What are the dates &amp; times you'll be available for interviewing?</li>
-          <li>Best number to contact you?</li>
+          <li>Interviews will be scheduled for [insert days here]. Please indicate any conflicts you may have on these days.</li>
+          <li>What is the best number with which to contact you regarding your interview?</li>
```
Comment (RobertWSaunders) [inline app/views/job_posting_questions/index.html.erb:47]: I'm not sure if "with which" sounds good here. I think "What is the best number to contact you at regarding your interview?" would sound better. What are your thoughts?

File: app/views/job_posting_questions/index.html.erb [hunk 2]
```diff
@@ -43,8 +43,8 @@
           Recommmended Questions:
         </p>
         <ul>
-          <li>What are the dates &amp; times you'll be available for interviewing?</li>
-          <li>Best number to contact you?</li>
+          <li>Interviews will be scheduled for [insert days here]. Please indicate any conflicts you may have on these days.</li>
```
Comment (RobertWSaunders) [inline app/views/job_posting_questions/index.html.erb:46]: This isn't really a question, can we rephrase it to be one. I think it makes sense as this is the "Recommended Questions:" card.

**Review by RobertWSaunders — CHANGES_REQUESTED** [review]
A couple of questions with this! Also, a few other things I noticed, see below:

- Make sure to fill out the entire template, regardless of the size of change being proposed.
- Have you run the application locally to see what these changes look like?
  - Because you changed one of the lines to be much longer the UI could look all messed up.
- I noticed that you have already committed this to your projects `dev` branch. It would have been better to make a feature branch as we discussed, once you have a feature branch you would request to merge that branch into this `dev` branch. Here you are merging your `dev` into my `dev` which is quite bad. In a large application with many collaborators `dev` would be changing frequently so this PR would explode will all of those new commits as well. I suggest reading about the feature branch Git workflow, I found [this](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) piece.
- Use to the imperative in your commit message. Instead of "Minor HTML updates to index.html.erb" maybe write "Update job posting questions index page copy". Using the imperative in commit messages is good practice and is something that Git itself does. i.e. "Merge pull request ...". Read more about it [here](https://chris.beams.io/posts/git-commit/).

When you address these comments, please do an interactive rebase and force push to rewrite the history of your commit instead of committing on top of your change. If you don't know what a rebase is or even an interactive rebase is, read about it [here](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase) and more specifically about rewriting commit history [here](https://www.atlassian.com/git/tutorials/rewriting-history). This is something that you'll want to get good at as in the industry it's pretty much mandatory. If you don't rebase then your commit history becomes way too bloated. This is something I learned this summer and this project in the past has done a really bad job at, moving forward I'd like to change that. Refer to the image below to see exactly what we are trying to avoid:

screen shot 2018-09-27 at 12 02 24 pm

Ideally these commits should all be squashed into one commit because they are doing the same thing. In my summer job a PR would not be allowed to be merged if it had more than five commits. Each commit should serve a distinct purpose and not overlap with changes in another commit. So in the example above, having three commits that are doing the same thing is terrible. Beyond having a good commit history, having distinct commits is useful for many other reasons. Some being, you are able to see individual changes on a per commit basis and know exactly what those changes are doing, you can more easily rewrite history if a commit encapsulates a change related to one change and so many more. I'm in the process of writing a blog post all about this stuff because not many students fully grasp these concepts but from my experience in the industry they are very important.

I don't mean to roast this PR, I just want to knowledge share as much as I can! Let me know if you have any questions and I am happy to meet up, I'll be releasing my blog post on Sunday so maybe you can give it a read and then if you are still confused we can meet up! Otherwise feel free to take a shot at addressing the things mentioned by yourself!

**Comment by maxkaran** [comment]
Ok! Thanks for all the feedback, I will close this pull request and try again!
